src/containers/entitysetsearch/EntitySetSearchResults.js

```diff
@@ -599,6 +599,14 @@ export default class EntitySetSearchResults extends React.Component {
     });
   }
 
+  removeDuplicates = (list1 :List<any>, list2 :List<any>) => {
+    const valueSet = Immutable.Set();
+    list1.concat(list2).forEach((val) => {
+      valueSet.add(val);
```

Comment:
Since `valueSet` is immutable, you're returning an empty set. Each `.add()` returns a new object that contains the added element. You want `let valueSet`, and then `valueSet = valueSet.add(val); `

Reply:
oh yeah that's weird, why was this working locally then? :/

Reply:
Are you sure it was working locally? Have you stepped through with the debugger?

Reply:
not with the debugger, but I had confirmed it was working locally. but anyway, I updated it.